pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -350,15 +333,15 @@ in
     postPatch = ''
       sed -i -e 's/require "health"/vim.health/' lua/clipboard-image/health.lua
     '';
-    nvimRequireCheck = "clipboard-image";
   };
 
   cmake-tools-nvim = super.cmake-tools-nvim.overrideAttrs {
     dependencies = [ self.plenary-nvim ];
-    nvimRequireCheck = "cmake-tools";
   };
 
   cmd-parser-nvim = super.cmd-parser-nvim.overrideAttrs {
+    # Has cmd-parser.init.test matched from lua/cmd-parser/init.test.lua
```

Comment:
Maybe ignore all files that end in `.test.lua`? (Not a requirement, just a suggestion)

Reply:
Yeah, I'll do another Pr for that that targets staging.